codegen/src/main/scala/bsp/codegen/ToLines.scala

```diff
@@ -12,6 +12,8 @@ object ToLines {
 
   implicit def flattenedLines[A: ToLines]: ToLines[List[A]] = (a: List[A]) => a.flatMap(ToLines(_))
 
-  implicit def linesToLInes: ToLines[Lines] = (a: Lines) => a.get
+  implicit def linesToLines: ToLines[Lines] = (a: Lines) => a.get
+
+  implicit def optionToLines[A: ToLines]: ToLines[Option[A]] = (a: Option[A]) => ToLines(a.toList)
```

Comment:
Maybe I'm missing the usage, but isn't this unused?

Reply:
yeah, that's a leftover from the previous implementation